src/Examples/Flowsheets/power_generation/rsofc/rsofc_soec_example.ipynb

```diff
@@ -0,0 +1,2131 @@
+      "  88  2.6470427e+06 5.80e+02 3.12e+03  -2.5 1.12e+02  -3.8 1.00e+00 1.00e+00h  1\n",
+      "  89  2.6470338e+06 1.73e+03 8.69e+03  -2.5 4.65e+01  -2.5 6.23e-01 1.00e+00h  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "  90  2.6470393e+06 6.42e+02 3.63e+03  -2.5 1.52e+02    -  9.81e-01 1.00e+00h  1\n",
+      "  91  2.6470267e+06 7.42e+01 3.20e+03  -2.5 1.43e+01  -1.1 4.74e-01 1.00e+00h  1\n",
+      "  92  2.6470266e+06 7.39e+01 6.06e+03  -2.5 2.59e+02    -  1.00e+00 4.16e-03h  1\n",
+      "  93  2.6470178e+06 2.79e+02 3.99e+03  -2.5 2.37e+02    -  1.00e+00 1.00e+00h  1\n",
+      "  94  2.6469897e+06 4.26e+02 1.81e+03  -2.5 3.32e+01  -1.6 6.14e-01 1.00e+00f  1\n",
+      "  95  2.6469866e+06 4.26e+02 2.02e+03  -2.5 8.77e+02    -  7.62e-02 4.36e-03f  1\n",
+      "  96  2.6468246e+06 4.39e+03 1.64e+03  -2.5 1.18e+03    -  1.23e-01 1.68e-01f  1\n",
+      "  97  2.6468242e+06 4.38e+03 3.89e+03  -2.5 3.04e+02    -  1.00e+00 1.82e-03h  1\n",
+      "  98  2.6468302e+06 4.53e+02 1.43e+03  -2.5 2.58e+02    -  1.00e+00 1.00e+00h  1\n",
+      "  99  2.6468308e+06 1.41e+02 4.23e+02  -2.5 8.19e+02    -  9.19e-01 7.03e-01h  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      " 100  2.6468311e+06 1.33e+00 5.00e+02  -2.5 1.23e+00  -2.1 1.00e+00 1.00e+00h  1\n",
+      " 101  2.6468311e+06 1.23e-04 1.41e+03  -2.5 1.47e-01  -2.6 1.00e+00 1.00e+00h  1\n",
+      " 102  2.6468311e+06 2.56e-06 6.78e+02  -3.8 1.88e-01  -3.0 1.00e+00 1.00e+00f  1\n",
+      " 103  2.6468311e+06 8.96e+00 8.88e+02  -3.8 1.67e+03    -  7.58e-01 1.00e+00h  1\n",
+      " 104  2.6468311e+06 3.18e+02 6.05e+02  -3.8 5.97e+04    -  4.58e-01 1.70e-01h  2\n",
+      " 105  2.6468311e+06 2.04e+01 8.90e+02  -3.8 7.14e+02    -  7.13e-01 1.00e+00h  1\n",
+      " 106  2.6468311e+06 2.48e+02 3.03e+03  -3.8 3.24e+04    -  3.23e-01 1.68e-01h  2\n",
+      " 107  2.6468311e+06 3.92e+02 9.92e+03  -3.8 7.90e+03    -  5.63e-01 3.44e-01h  2\n",
+      " 108  2.6468311e+06 8.17e+01 2.68e+04  -3.8 9.32e+02    -  3.16e-01 1.00e+00H  1\n",
+      " 109  2.6468311e+06 2.06e+02 6.62e+02  -3.8 1.30e+03    -  1.00e+00 1.00e+00h  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      " 110  2.6468311e+06 5.86e+00 1.69e+01  -3.8 5.42e+02    -  1.00e+00 1.00e+00h  1\n",
+      " 111  2.6468311e+06 1.11e+02 2.43e+03  -3.8 6.80e+02    -  1.00e+00 1.00e+00h  1\n",
+      " 112  2.6468311e+06 8.16e-01 4.54e+00  -3.8 2.20e+02    -  1.00e+00 1.00e+00h  1\n",
+      " 113  2.6468311e+06 4.76e-02 1.20e+00  -3.8 1.14e+01    -  1.00e+00 1.00e+00h  1\n",
+      " 114  2.6468311e+06 3.10e-07 7.29e-06  -3.8 4.99e-02    -  1.00e+00 1.00e+00h  1\n",
+      " 115  2.6468311e+06 6.31e-04 2.08e-03  -5.0 1.90e+01    -  1.00e+00 1.00e+00f  1\n",
+      " 116  2.6468311e+06 1.79e-05 1.44e-01  -5.0 2.98e+02    -  1.00e+00 1.00e+00H  1\n",
```

Comment:
Minor point, but this solve is a bit ugly - I am a little worried it might prove to be fragile and cause failures later.